apps/balancer-tools/src/app/apr/pool/(components)/PoolOverviewCards.tsx

```diff
@@ -0,0 +1,25 @@
+import { formatDate } from "@bleu-balancer-tools/utils";
+
+import { formatNumber } from "#/utils/formatNumber";
+
+import OverviewCards from "../../(components)/OverviewCards";
+import { calculateAPRForPool } from "../../(utils)/calculateRoundAPR";
+import { Round } from "../../(utils)/rounds";
+
+export default async function PoolOverviewCards({
+  roundId,
+  poolId,
+}: {
+  roundId: string;
+  poolId: string;
+}) {
+  const { apr, tvl } = await calculateAPRForPool({ poolId, roundId });
```

Comment:
if you want to return more data from this function then don't bake in the name the expectation that you're going to only return one -- you could name it e.g. `calculatePoolStats`